lib/analytics.js

```diff
@@ -305,18 +370,18 @@ Analytics.prototype.track = function(event, properties, options, fn) {
   });
 
   // plan.
-  plan = events[event];
-  if (plan) {
-    this.log('plan %o - %o', event, plan);
-    if (plan.enabled === false) {
+  var trackPlan = trackPlans[event];
```

Comment:
`var eventPlan` maybe?

Reply:
I thought `trackPlans` made more sense since it's on `this.options.plan.track`.